RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -156,11 +156,13 @@ extension CloudBackupClient {
 				try? userDefaults.removeLastCloudBackup(for: profile.id)
 			}
 
-			let needsBackUp = profile.appPreferences.security.isCloudProfileSyncEnabled && profile.header.isNonEmpty
+			let isSynced = profile.appPreferences.security.isCloudProfileSyncEnabled
```

Comment:
I think `isSynced` is a misleading variable name, sounds like a sync status is *known to be in status 'synced'*, I'd call it isCloudSyncEnabled` or similar.

Reply:
Sure